pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -477,10 +477,13 @@ func (s *Server) createIstioRA(client kubelib.Client,
 	caCertFile := path.Join(ra.DefaultExtCACertDir, constants.CACertNamespaceConfigMapDataName)
 	certSignerDomain := opts.CertSignerDomain
 	_, err := os.Stat(caCertFile)
-	if err != nil && certSignerDomain == "" {
-		caCertFile = defaultCACertPath
-	} else {
-		caCertFile = ""
+	if err != nil {
+		log.Infof("CA cert file stat error: %v", err)
```

Comment:
nit: this seems like its logging an error. Would prefer to say "CA cert file not found, using caCertFile <xyz>"